prov/gni/src/gnix_ep.c

```diff
@@ -1084,6 +1084,142 @@ int gnix_ep_open(struct fid_domain *domain, struct fi_info *info,
 
 }
 
+static int __match_context(struct slist_entry *item, const void *arg)
+{
+	struct gnix_fab_req *req;
+
+	req = container_of(item, struct gnix_fab_req, slist);
+
+	GNIX_INFO(FI_LOG_EP_CTRL, "testing: req->context=%p context=%p item=%p\n",
+			req->user_context, arg, item);
```

Comment:
It should just be removed. I added it for testing purposes and it doesn't really serve much purpose.